safekeeper/src/wal_backup.rs

```diff
@@ -0,0 +1,218 @@
+
+use std::{path::PathBuf};
+
+use tokio::fs;
+use tokio::runtime::{Builder, Runtime};
+
+use lazy_static::lazy_static;
+use anyhow::{Result, ensure};
+
+use tokio::sync::watch::Receiver;
+use utils::{
+    lsn::Lsn,
+    zid::{ZTenantTimelineId},
+};
+
+use tracing::*;
+
+use crate::remote_storage::{LocalFs, RemoteStorage};
+use crate::{broker, SafeKeeperConf};
+
+lazy_static! {
+    static ref BACKUP_RUNTIME: Runtime = {
+        Builder::new_multi_thread()
+            .worker_threads(16)
```

Comment:
`16` looks magical, esp. considering that we're good with a bunch of separate single threaded runtimes in general: all `tokio::fs` operations spawn separate threads to deal with the i/o concurrently, the rest of the operations seem to be non-blocking, so we might just use a single threaded runtime for experiments for now.

Or add a configurable value. Or a comment at least.

Reply:
addressed